### PR TITLE
Refactor/colors

### PR DESCRIPTION
**Pull request by irmakerdem**
What does this PR do?
- Adds color to the dashboard
- Distinguishes labels from values by bolding and italicizing 

Where should your reviewer start?
- index.html line 9
- line 2 styles.css
- line 71 scripts.js

What (if any) features are you implementing?
- Attempt to make website colorblind-friendly

What (if anything) did you refactor?
- Removed comments

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -30,33 +30,33 @@ <h2 class="welcome-text" id="welcomeText">Welcome, name!</h2>
       <div class="right-and-bottom">
         <div class="right-side">
           <section class="steps-card">
-            <h3>Steps</h3>
-            <p class="step-data" id="yourStepGoal">Your Step Goal:</p>
+            <h3>👟 Steps 🥾</h3>
```
Comment (catlynbowles) [inline dist/index.html:33]: LOVE 🐶

File: src/css/styles.css [hunk 2]
```diff
@@ -40,6 +46,18 @@ h1, h5, .info-card, .steps-card, .sleep-card, .hydration-card {
   text-align: center;
 }
 
+h3 {
```
Comment (catlynbowles) [inline src/css/styles.css:49]: love the colors! 💅🏼

**Review by catlynbowles — APPROVED** [review]

File: dist/index.html (hunk 1 above)
Comment (ASands17) [reply dist/index.html:33]: This looks awesome! 🌈

File: src/scripts.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ function displayHydrationInfo(newUser) {
 }
 
 function displayIdCardInfo(newUser) {
-  idText.innerText += ` ${newUser.id}`;
+  // idText.innerText += ` ${newUser.id}`;
```
Comment (ASands17) [inline src/scripts.js]: This looks much better. Nice work. ✨

**Comment by ASands17** [comment]
Great work on these refactors! I think our dashboard is much more cute and readable now. 👑

**Pull request merged**